server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -0,0 +1,736 @@
+Again, you may need to check all the caches and bring them online.
+
+Once the sites are put online, it's usually good to:
+* Do the link:statetransfer[state transfer].
+* Manually link:clearcache[clear the Keycloak caches]. 
+
+
+[[statetransfer]]
+==== State transfer
+
+State transfer is a manually required step. JDG does not do this automatically, for example during split-brain, it is only the admin who may decide which site has preference and hence if state transfer needs to be done bidirectionally between both sites or just unidirectionally, as in only from `site1` to `site2`, but not from `site2` to `site1`. 
+
+A bidirectional state transfer will ensure that entities which were created *after* split-brain on `site1` will be transferred to `site2`. This is not an issue as they do not yet exist on `site2`. Similarly, entities created *after* split-brain on `site2` will be transferred to `site1`. Possibly problematic parts are those entities which exist *before* split-brain on both sites and which were updated during split-brain on both sites. When this happens, one of the sites will *win* and will overwrite the updates done during split-brain by the second site.
+
+Unfortunately, there is no any universal solution to this. Split-brains and network outages are just state, which is usually impossible to be handled 100% correctly with 100% consistent data between sites. In the case of {project_name}, it typically is not a critical issue. In the worst case, users will need to re-login again to their clients, or have the improper count of loginFailures tracked for brute force protection. See the JDG/JGroups/Infinispan docmentation for more tips how to deal with split-brain.
+
+The state transfer can be also done on the JDG side through JMX. The operation name is `pushState`. There are few other operations to monitor status, cancel push state, and so on. 
+More info about state transfer is available in the link:https://access.redhat.com/documentation/en-us/red_hat_jboss_data_grid/7.1/html/administration_and_configuration_guide/set_up_cross_datacenter_replication#state_transfer_between_sites[JDG docs].
+
+
+[[clearcache]]
+==== Clear caches
+
+After split-brain it is safe to manually clear caches in the {project_name} admin console. This is because there might be some data changed in the database on `site1` and because of the event, that the cache should be invalidated wasn't transferred during split-brain to `site2`. Hence {project_name} nodes on `site2` may still have some stale data in their caches.
+
+To clear the caches, see {adminguide_clearcache_link}[{adminguide_clearcache_name}].
+
+When the network is back, it is sufficient to clear the cache just on one {project_name} node on any random site. The cache invalidation event will be sent to all the other {project_name} nodes in all sites. However, it needs to be done for all the caches (realms, users, keys).    
```

Comment:
Reference to where to find in in admin console would help (hint: server_admin/topics/realms/cache.adoc file)